noir-projects/noir-contracts/contracts/nft_contract/src/main.nr

```diff
@@ -157,12 +164,12 @@ contract NFT {
             TRANSIENT_STORAGE_SLOT_PEDERSEN_INDEX
         );
 
-        NFT::at(context.this_address())._store_point_in_transient_storage(hiding_point, slot).enqueue(&mut context);
+        NFT::at(context.this_address())._store_point_in_transient_storage(hiding_point.inner, slot).enqueue(&mut context);
```

Comment:
Is this because we cannot generate Serialize trait for the hiding point?

(This will change a lot with partial notes log so makes sense to leave it as is now.)